tableaudocumentapi/field.py

```diff
@@ -199,4 +200,9 @@ def _read_description(xmldata):
         if description is None:
             return None
 
-        return u'{}'.format(ET.tostring(description, encoding='utf-8'))  # This is necessary for py3 support
+        description_string = ET.tostring(description, encoding='utf-8')
+        # Format expects a unicode string so in Python 2 we have to do the explicit conversion
+        if sys.version_info[0] == 2:
+            description_string = description_string.decode('utf-8')
+
+        return u'{}'.format(description_string)  # This is necessary for py3 support
```

Comment:
Hello.

Yes it makes sense to check the type instead of the Python version.

I'll sign later the CLA, do the rebase and fix the commit to get rid of sys.

Thanks,
Miguel.